spec/requests/users_spec.rb

```diff
@@ -28,17 +28,6 @@
     end
   end
 
-  # We do NOT allow the creation of arbitrary users. New users should only be created
-  # via CAS authentication.
-  describe "GET /new" do
-    it "redirects to CAS" do
-      get "/users/new"
-      expect(response.code).to eq "302"
-      redirect_location = response.header["Location"]
-      expect(redirect_location).to eq "http://www.example.com/sign_in"
-    end
-  end
-
```

Comment:
I don't think we need this test. The authenticate method will force the process to go through CAS. 

Is it possible that we have a user with a netid of `new`?